crev-lib/src/id.rs

```diff
@@ -82,18 +83,18 @@ impl LockedId {
             Self::weak_passphrase_config()
         };
 
-        let pwsalt = random_vec(32);
+        let pwsalt = random_vec(16);
         let pwhash =
             argon2::hash_raw(passphrase.as_bytes(), &pwsalt, &config).map_err(Error::Passphrase)?;
 
-        let mut siv = miscreant::Aes256SivAead::new(&pwhash);
+        let siv = aes_siv::Aes256SivAead::new(pwhash.as_slice().into());
```

Comment:
I believe this can be made backwards-compatible if we used `aes_siv::siv::Aes256SivAead` and `encrypt_in_place_detached()` [the way miscreant did](https://github.com/miscreant/miscreant.rs/blob/53b3ae9fabe049ce2a6341d10aa359546b03c734/src/aead.rs#L166). Miscreant depended on an older version of `ctr` crate which provided `Ctr128`. Glancing at [the PR that removed it](https://github.com/RustCrypto/stream-ciphers/pull/195/files), it appears that `Ctr128BE` is a compatible replacement.

I can prepare a PR if you want, except I don't know how to test that keys are compatible.

Reply:
Yes, please make a PR. There's a unit test for the keys, so `cargo test -all` is sufficient.

Reply:
Done: #414.